src/xdocs/google_style.xml

```diff
@@ -2290,8 +2290,8 @@
                   </a>
                 </td>
                 <td>
-                  <a href="styleguides/google-java-style-20180523/javaguide.html#s7.3.1-javadoc-exception-self-explanatory">
-                    7.3.1 Exception: self-explanatory methods</a>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s7.3.1-javadoc-exception-self-explanatory">
+                    7.3.1 Exception: self-explanatory members</a>
```

Comment:
Renamed the section name, detected at: https://github.com/checkstyle/checkstyle/pull/14901#discussion_r1660928171

Reply:
also updated the test class name according to this